docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -113,14 +123,35 @@ Users can install `scikits.odes <https://github.com/bmcage/odes>`__ to utilize i
    The ``scikits.odes`` solver is not supported on Python 3.12 yet. Please refer to https://github.com/bmcage/odes/issues/162.
    There is support for Python 3.8, 3.9, 3.10, and 3.11.
 
-.. tab:: GNU/Linux
+.. tab:: Debian/Ubuntu/Linux Mint
 
    In a terminal, run the following commands:
 
    .. code:: bash
 
       apt-get install libopenblas-dev
-      pip install wget cmake
+      pybamm_install_odes
```

Comment:
We should install CMake here, whether through `apt-get` or any other method like `pipx` (though `apt-get` is preferred).